ui/src/api/index.ts

```diff
@@ -32,11 +32,11 @@ Hydra.resources.factory.addMixin(JobCollectionMixin)
 Hydra.resources.factory.addMixin(...ShapeBundle)
 
 // Inject the access token in all requests if present
-Hydra.defaultHeaders = () => {
+Hydra.defaultHeaders = ({ uri }) => {
   const headers = new Headers()
 
   const token = store.state.auth.access_token
-  if (token) {
+  if (token && uri.startsWith(rootURL)) {
```

Comment:
The `{ uri }` param is a new feature in alcaeus.

Prevents adding of auth header to requests outside our API